scripts/create-a-m-o/functions.js

```diff
@@ -0,0 +1,221 @@
+
+  fs.writeFileSync(
+    `${BASE_FOLDER}/package.json`,
+    JSON.stringify(templateJson, null, 2),
+    'utf8',
+  );
+
+  fs.writeFileSync(
+    `${BASE_FOLDER}/README.md`,
+    outdent`
+    # ${compTitle}
+
+    TODO Description
+    `,
+    'utf8',
+  );
+
+  fs.writeFileSync(
+    `${BASE_FOLDER}/index.scss`,
+    outdent`
+      .${type}-${fileName} {
+        display: block;
+      }
+    `,
+    'utf8',
+  );
+
+  fs.writeFileSync(
+    `${BASE_FOLDER}/index.react.js`,
+    outdent`
+    import withReact from '../../../utils/with-react';
+    import ${className} from './index';
+
+    export default createElement => ({
+      /* props here, same as in the constructor of index.js */
+      children,
+    }) =>
+      withReact(createElement)(
+        ${className}.tagName,
+        {
+          /* props here, same as in the constructor of index.js */
+        },
+        children
+      );
+
+    `,
+    'utf8',
+  );
+
+  fs.writeFileSync(
+    `${BASE_FOLDER}/story.js`,
+    outdent`
+    /* global document */
+    import { storiesOf } from '@storybook/html';
+    import './index';
+    import { withMarkdown } from '../../../../.storybook/addons/markdown';
+    import Readme from './README.md';
+
+    storiesOf('${titleMap[type]}/${compTitle}', module)
+      .addDecorator(withMarkdown(Readme))
+      .add('${compTitle} - default', () => '<axa-${fileName}>Some children</axa-${fileName}>')
+    `,
+    'utf8',
+  );
+
+  fs.writeFileSync(
+    `${BASE_FOLDER}/ui.test.js`,
+    outdent`
+    import { Selector } from 'testcafe';
+
+    fixture('${compTitle} - basic functionality').page('http://localhost:9999/iframe.html?id=${titleMap[type].toLowerCase()}-${fileName}--${fileName}-default');
+
+    const TAG = 'axa-${fileName}';
+    const CLASS = '.${type}-${fileName}';
+
+    test('should render ${fileName}', async t => {
+      const $axaElem = await Selector(TAG);
+      await t.expect($axaElem.exists).ok();
+      const $axaElemShadow = await Selector(() => document.querySelector('axa-${fileName}').shadowRoot);
+      const $axaElemShadowEl = await $axaElemShadow.find(CLASS);
+      await t.expect($axaElemShadowEl.exists).ok();
+    });
+    `,
+    'utf8',
+  );
+
+  fs.writeFileSync(
+    `${BASE_FOLDER}/index.js`,
+    outdent`
+    import '@webcomponents/webcomponentsjs';
+    import { LitElement, html, css, unsafeCSS } from 'lit-element';
+
+    /* eslint-disable import/no-extraneous-dependencies */
+    import defineOnce from '../../../utils/define-once';
+    import styles from './index.scss';
+
+    class ${className} extends LitElement {
+      static get tagName() {
+        return 'axa-${fileName}';
+      }
+
+      static get styles() {
+        return css\`
+          \${unsafeCSS(styles)}
+        \`;
+      }
+
+      static get properties() {
+        // Here you can define all properties and theier type
```

Comment:
Better -> `// Define properties and types`

Reply:
Done